Lib/urllib/parse.py

```diff
@@ -38,29 +39,36 @@
            "DefragResult", "ParseResult", "SplitResult",
            "DefragResultBytes", "ParseResultBytes", "SplitResultBytes"]
 
-# A classification of schemes ('' means apply by default)
-uses_relative = ['ftp', 'http', 'gopher', 'nntp', 'imap',
+# A classification of schemes.
+# We use '' as the scheme value for default scheme in urlsplit and urlparse.
+
+uses_relative = ['', 'ftp', 'http', 'gopher', 'nntp', 'imap',
```

Comment:
The comment applies to all three lists, `uses_relative`, `uses_netloc` and `uses_params`.
All of them have '' (empty string listed as supported scheme, because that is the default 
```def urlsplit(url, scheme='', allow_fragments=True):```
and
```def urlparse(url, scheme='', allow_fragments=True):```

And the current comment is:
```# A classification of schemes ('' means apply by default)``` 

* I think, the existing comment is confusing, and thus, this change is to make it a bit clear.